### DISCUSSION TITLE
Feature Request: swap 'Commit' and 'Diff' tabs

### DISCUSSION THREAD

**Discussion by igroglaz** [gitextensions/gitextensions#9657] [Ideas] [answered]
## Feature description

It will be awesome to add option to "Settings" which will provide possibility to swap this two tabs:

![swap](https://user-images.githubusercontent.com/39904162/137600010-cc3276a6-f9e3-4f9a-a720-342eea7fc79a.png)

...so when user click on any commit in the list - user by default will see not the commit description, but files which were changed.

## Environment

- Git Extensions 3.5.4.12724
- Build 65f01f39982ccb121c4df7d4b00b506939553958
- Git 2.33.0.windows.2
- Microsoft Windows NT 10.0.19043.0
- .NET Framework 4.8.4420.0
- DPI 144dpi (150% scaling)

**Comment by gerhardol** [comment, marked as answer]
A change like that will bring down rage from many users, even if it is what I would prefer myself. (The commit info is mostly in the revision grid anyway.)
You can rearrange so the commit info is in the revision grid instead
![image](https://user-images.githubusercontent.com/6248932/137601213-1164e773-525d-4fe2-9417-5c7a04de1d35.png)

**Comment by mstv** [comment]
I'd rather make `Commit right of graph` the default setting:

![image](https://user-images.githubusercontent.com/36601201/137601222-8e96036a-8e3e-4d17-b350-2c48f5cbece2.png)

**Reply by RussKie** [reply]
This layout has never grown on me. It is great that we like different layouts and the app support those.

**Comment by igroglaz** [comment]
Cool, thanks, folks! I'll use `Commit right of graph`.

Actually in this case it will be fun to be able to hide Commit tab from UI.. Cause I use this layout now:
![image](https://user-images.githubusercontent.com/39904162/137626782-4bc60ff1-2bac-4eaf-8961-009ad44bb7b5.png)
..so I got to hide commit tab manually and its 'ears' stuck from right side 👂  which is kinda untidy 🤣
